### ISSUE TITLE
Drag and drop functionality unavailable to keyboard or touchscreen

### ISSUE THREAD

**Issue by karlgroves** [WordPress/gutenberg#15316]
Drag and drop functionality unavailable to keyboard or touchscreen

-   **Severity:** Low
-   **Affected Populations:**
    -   Blind
    -   Motor Impaired
-   **Platform(s):**
    -   All / Universal
-   **Components affected:**
    -   Edit Media

#### Issue description

The Edit Media interface includes the ability to re-arrange the boxes by
drag and drop, however this can only be achieved by mouse users, since
the functionality does not support touch or keyboard users.

Users of touchscreens and keyboards cannot re-order the boxes. Speech
recognition users may be able to if they can manipulate the mouse with
speech (such as using Mouse Move), however this is onerous and
difficult.

#### Remediation Guidance

Allow keyboard and touchscreen users the ability to re-arrange the
boxes. This can either be done with dedicated buttons, such as are used
within Gutenberg for moving Blocks; or it could be done by extending
drag and drop support to keyboard and touch users (a good example of
which is dragondrop: <https://schne324.github.io/dragon-drop/demo/)>.

#### Relevant standards

-   [2.1.1 Keyboard
    (Level A)](https://www.w3.org/TR/WCAG20/#keyboard-operation-keyboard-operable)

**Note:** This issue may be a duplicate with other existing accessibility-related bugs in this project. This issue comes from the Gutenberg accessibility audit, performed by [Tenon](https://www.tenon.io) and funded by [WP Campus](https://wpcampus.org/). This issue is GUT-80 in Tenon's report

**Comment by melchoyce** [comment]
Screenshot from the report:

![image](https://user-images.githubusercontent.com/2846578/57182776-395bb680-6e58-11e9-9962-4722c9f98625.png)

Note: Since this is a core Media issue, not Gutenberg, we'll want to move this issue to Trac.

**Comment by afercia** [comment]
> The Edit Media interface 

Original issue description and screenshot are available at page 142 of [the complete technical report PDF](https://documents.wpcampus.org/gutenberg/audit/Gutenberg_Report.pdf). However, the screenshot appears to be related to the edit post screen in the classic editor. Regardless, this issue should be moved to Trac as all the meta boxes in the core admin pages can't be reordered with keyboard.

**Comment by anevins12** [comment]
Moved to Trac https://core.trac.wordpress.org/ticket/47135

**Comment by afercia** [comment]
Closing as moved to Trac.
